overlord/devicestate/devicemgr.go

```diff
@@ -160,6 +160,8 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	// config assets are assumed to be always backwards compatible.
 	runner.AddHandler("update-managed-boot-config", m.doUpdateManagedBootConfig, nil)
 
```

Comment:
I don't think we want this blank line, maybe just have a trivial comment if the issue with having nothing is the confusion with the previous comment